### DISCUSSION TITLE
How to solve prefer_const_constructor

### DISCUSSION THREAD

**Discussion by surathadk** [community/community#28036] [Other Feature Feedback, Questions, & Ideas] [answered]
Dear Community:

After flutter upgrade to 3.0 , it asks to add const to constructor, I wonder if there is any way to automatically add const .

**Comment by sthakrishnakumar** [comment, marked as answer]
Hello Dear,
               I acknowledged your issue and the solution to your issue is here:
You can add following code to the settings.json file 

 "editor.codeActionsOnSave": {
            "source.fixAll":true
        },

**Reply by surathadk** [reply]
Thank so much @sthakrishnakumar
